### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ivanfifan**
[DEMO LINK](https://Ivanfifan.github.io/layout_calendar/)
[TEST REPORT LINK](https://Ivanfifan.github.io/layout_calendar/report/html_report/)

There is a problem, that I can't deploy project to GitHub pages. I ask for help in fe_chat, but no one can solve this problem for a long time

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,53 @@
+$cell-size: 100px;
+$start-day: "mon" 0 0, "tue" 1 3, "wed" 2 6, "thu" 3 9, "fri" 4 12, "sat" 5 15, "sun" 6 18;
```
Comment (nazarmatsevych) [inline src/styles/main.scss]: Create a separate file for variables

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Code looks good but demo link doesn't work, try to rerun "npm run deploy" command or ask for help in fe_chat

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-size * 7+20;
```
Comment (Anastasiia-Svintsova) [inline src/styles/main.scss]: Don't use mystery numbers. Move 7 and 20 to variables

File: src/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+$cell-size: 100px;
+$start-day: "mon" 0 0, "tue" 1 3, "wed" 2 6, "thu" 3 9, "fri" 4 12, "sat" 5 15, "sun" 6 18;
```
Comment (Anastasiia-Svintsova) [inline src/styles/variables.scss]: For what purposes do we need to use the second number for each item? You can simplify it and calculate the necessary values 
```suggestion
$start-day: "mon" 1,  "tue" 2,  "wed" 3,  "thu" 4,  "fri" 5,  "sat" 6,  "sun" 7;
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Hey! Seems the project is not deployed. Ask questions in the chat if you need a hint 
image

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your demo link isn't working

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Good code! Unfortunately, your demo link doesn't work: https://mate-academy.github.io/layout_task-guideline/#important use this instruction (18) to deploy your site

**Pull request closed without merging**